addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9553,7 +9553,15 @@ msgctxt "#19254"
 msgid "Guide update failed for channel"
 msgstr ""
 
-#empty strings from id 19255 to 19256
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
```

Comment:
Please add a short explanation for the purpose of this string. <code>#. blah blah</code>